src/Analysis/SharedUtils/TeamProject/TeamProjectModal/TeamProjectModal.jsx

```diff
@@ -42,6 +59,32 @@ const TeamProjectModal = ({
       </Modal>
     );
   }
+  if (isWarningModalOpen === true) {
+    return (
+      <Modal
```

Comment:
this is now a modal in a modal. Maybe consider making the warning be its own modal component.

Reply:
hmm not sure what you mean, it is still returning a team project modal like before, but if the warning flag is triggered, it's returning different component/message inside it?

Reply:
I see now.... Maybe the confusing part is that the TeamProjectModal.jsx is actually different modals, depending on the state variables. I'm fine with this for now, given the plans to refactor the frontend code.